doc/source/user_guide/scale.rst

```diff
@@ -217,190 +217,10 @@ require too sophisticated of operations. Some operations, like :meth:`pandas.Dat
 much harder to do chunkwise. In these cases, you may be better switching to a
 different library that implements these out-of-core algorithms for you.
 
-.. _scale.other_libraries:
-
-Use Dask
---------
+Use Other Libraries
+-------------------
 
 pandas is just one library offering a DataFrame API. Because of its popularity,
```

Comment:
This paragraph was probably ok when Dask was discussed in detail here, but I think now it does a very poor job at pointing out to the ecosystem.

A bit of context: This is the documentation for scaling pandas (using pandas with data too big to fit in memory, or to process with a single computer). Besides what's explained above of this section, we want users to know that there are a set of libraries such as PySpask, Dask and Modin that implement an API almost identical to the pandas one, but run in clusters. And that they can find more information in the ecosystem page.

Do you mind trying to rephrase this section in a way that is helpful for users to understand this @yukikitayama ?

Thank you very much for the work here.